libs/payments/customer/src/lib/customer.manager.ts

```diff
@@ -69,6 +73,13 @@ export class CustomerManager {
     return customer;
   }
 
+  /**
+   * Reload the customer data to reflect a change.
+   */
+  async changed(uid: string) {
+    await this.profileClient.deleteCache(uid);
```

Comment:
It looks like there was some functionality around notifications (logs, emails, etc.) that no longer exist, is that intentional? (I see the commented-out portions, were those implemented/resolved in the linked tickets?)

Reply:
Goooood question... 

`push` is no longer required (see [ticket](https://mozilla-hub.atlassian.net/browse/FXA-9276) - learned this after accidentally creating a duplicate ticket 😅 ), so we don't need to implement that line nor the line creating `devices`. 

I don't believe we need the line to log via `notifyAttachedServices` - I'll ask in the team channel just to be sure!